src/Instances.js

```diff
@@ -59,15 +62,15 @@ const filterConfig = [
 ];
 
 const searchableIndexes = [
-  { label: 'All (title, contributor, identifier)', value: 'all', queryTemplate: 'title="%{query.query}" or contributors adj "\\"name\\": \\"%{query.query}\\"" or identifiers adj "\\"value\\": \\"%{query.query}\\""' },
-  { label: 'Barcode', value: 'item.barcode', queryTemplate: 'item.barcode=="%{query.query}"' },
-  { label: 'Instance ID', value: 'id', queryTemplate: 'id="%{query.query}"' },
-  { label: 'Title', value: 'title', queryTemplate: 'title="%{query.query}"' },
-  { label: 'Identifier', value: 'identifier', queryTemplate: 'identifiers adj "\\"value\\": \\"%{query.query}\\""' },
-  { label: '- ISBN', value: 'isbn', queryTemplate: 'identifiers == "*\\"value\\": \\"%{query.query}\\", \\"identifierTypeId\\": \\"<%= identifierTypeId %>\\""' },
-  { label: '- ISSN', value: 'issn', queryTemplate: 'identifiers == "*\\"value\\": \\"%{query.query}\\", \\"identifierTypeId\\": \\"<%= identifierTypeId %>\\""' },
-  { label: 'Contributor', value: 'contributor', queryTemplate: 'contributors adj "\\"name\\": \\"%{query.query}\\""' },
-  { label: 'Subject', value: 'subject', queryTemplate: 'subjects="%{query.query}"' },
+  { label: 'ui-inventory.search.all', value: 'all', queryTemplate: 'title="%{query.query}" or contributors adj "\\"name\\": \\"%{query.query}\\"" or identifiers adj "\\"value\\": \\"%{query.query}\\""' },
+  { label: 'ui-inventory.barcode', value: 'item.barcode', queryTemplate: 'item.barcode=="%{query.query}"' },
+  { label: 'ui-inventory.instanceId', value: 'id', queryTemplate: 'id="%{query.query}"' },
+  { label: 'ui-inventory.title', value: 'title', queryTemplate: 'title="%{query.query}"' },
+  { label: 'ui-inventory.identifier', value: 'identifier', queryTemplate: 'identifiers adj "\\"value\\": \\"%{query.query}\\""' },
+  { label: 'ui-inventory.search.isbn', value: 'isbn', queryTemplate: 'identifiers == "*\\"value\\": \\"%{query.query}\\", \\"identifierTypeId\\": \\"<%= identifierTypeId %>\\""' },
+  { label: 'ui-inventory.search.issn', value: 'issn', queryTemplate: 'identifiers == "*\\"value\\": \\"%{query.query}\\", \\"identifierTypeId\\": \\"<%= identifierTypeId %>\\""' },
```

Comment:
Put the "-" indentation of ISBN and ISSN here instead of in the translation string.

Reply:
@dltj, that will get messy. Currently, the entire `label` value is passed as a parameter to `intl.formatMessage()` so adding a prefix like `- ` is not easily supported without changes in a few different places. It's certainly possible, just not trivial.

Would an acceptable approach be for _all_ the `label` values to use `ui-inventory.search...` prefixes? Or is the the feeling that the prefix is formatting, not a translation, and it really doesn't belong in the translation files?

Reply:
@zburke It looks like we can try to use the solution like this #601 to handle this case. What do you think about it?

Reply:
@zburke : Oh, nuts.  From a translator's point of view, this issue is similar to the challenge we worked through last year where required fields had the "*" appended to them in the translations strings and we moved that to code.  @YevheniiMaltsev proposed what looks like an interesting approach in #601, but I'm not well versed enough in JavaScript to sign off of the approach.

Reply:
#601 LGTM